tests/unit_tests/main_test.cpp

```diff
@@ -131,7 +131,7 @@ int main(int argc, char** argv) {
 #ifdef __HIPSYCL__
                         if (dev.is_accelerator())
 #else
-                        if (!dev.is_accelerator())
+                    if (!dev.is_accelerator())
```

Comment:
Looks like this part is not aligned with "if" statements under other macro.

Reply:
Thank you! I think `clang-format` got confused. We will need to fix this or add an exception.